src/basic/cgroup-util.c

```diff
@@ -2588,10 +2588,16 @@ int cg_enable_everywhere(CGroupMask supported, CGroupMask mask, const char *p) {
                         continue;
 
                 n = cgroup_controller_to_string(c);
+
                 {
                         char s[1 + strlen(n) + 1];
 
-                        s[0] = mask & bit ? '+' : '-';
+                        bool enable = mask & bit;
+                        if (!enable && delegate) { /* shouldn't remove controllers on delegated unit */
+                            continue;
```

Comment:
nitpick: Indent by 8 spaces (not 4, as you have here.) Also, no braces for a block with a single command. See [CODING_STYLE](https://github.com/systemd/systemd/blob/master/doc/CODING_STYLE) for more details.

```c
        if (!enable && delegate) /* shouldn't remove controllers on delegated unit */
                continue;
```

Reply:
Will fix.